deeptools/bamCompare.py

```diff
@@ -305,7 +305,7 @@ def main(args=None):
                                      verbose=args.verbose
                                      )
 
-    wr.run(FUNC, func_args, args.outFileName, format=args.outFileFormat, smooth_length=args.smoothLength)
+    wr.run(FUNC, func_args, args.outFileName, format=args.outFileFormat, smoothLength=args.smoothLength)
```

Comment:
Here is a bit of history: first deeptools programs used camel case, then I learned about the python style guidelines (Devon's dreaded PEP8) that do not recommend to use the camel case. So after some point is stopped using camel case. You can tell new code from old code by this.  However, for the command line arguments I didn't make that change to keep some consistency.